code/modules/robotics/bot/guardbot.dm

```diff
@@ -406,47 +435,56 @@
 	emag_act(var/mob/user, var/obj/item/card/emag/E)
 		if(!user || !E) return 0
 
+		src.shotcount = 2
 		if (src.idle || !src.on)
-			boutput(user, "You show \the [E] to [src]! There is no response.")
-		else
+			if (!src.emagged)
+				boutput(user, "You wave \the [E] in front of [src]'s blank screen. It responds with a small puff of smoke.")
+			else
+				boutput(user, "You wave \the [E] in front of [src]'s blank screen. It doesn't seem to respond.")
+		else if (!src.emagged)
 			if (E.icon_state == "gold")
 				boutput(user, "You show \the [E] to [src]! They are super impressed!")
 				SPAWN_DBG(1 SECOND)
 					boutput(user, "Like, really REALLY impressed.  They probably think you're some kind of celebrity or something.")
-					sleep(1 SECOND)
-					boutput(user, "Or the president. The president of space.")
+					SPAWN_DBG(1 SECOND)
+						boutput(user, "Or the president. The president of space.")
+						SPAWN_DBG(1 SECOND)
+							boutput(user, "In fact they're so impressed that it shorts out their Spacelaw circuits![pick("", " Whoops.")]")
```

Comment:
why are these in spawns

Reply:
Beats me, they were like that when I got there.

Would sleep() work better? I'm just worried about using sleep() because I'm not *super* sure which procs are called by things that would have a *fit* if there's a sleep() in there somewhere.

Reply:
Don't stack spawns. sleep is safe to use inside a spawn, looks better and has arguably less overhead (or maybe not).